packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -31,13 +35,30 @@ const getNextIndex = (array, currentIndex, key) => {
 };
 
 const Columns = ({
+  setVisibleColumns,
+  getVisibleColumnsCount,
   filterString,
   columns,
   setColumnsObject,
   onSelectColumn,
   instructionsLabel = 'Press space bar to toggle drag drop mode, use arrow keys to move selected elements.',
   disabledInstructionsLabel = 'Reordering columns are disabled because they are filtered currently.',
+  selectAllLabel = 'Column name',
 }) => {
+  // const getVisibleColumnRatio = () => {
+  //   let result = 0;
+
+  //   for (let i = 0; i < columns.length; i++) {
+  //     if (columns[i].isVisible == true) {
+  //       result++;
+  //     }
+  //   }
+  //   return result;
+  // };
+
+  // const setVisibleColumnsChild = () => {
+  //   this.setVisibleColumns(getVisibleColumnRatio());
+  // };
```

Comment:
commented out code should really be cleaned out (there are of course exceptions).